src/closure-types.ts

```diff
@@ -189,6 +189,18 @@ function convertArray(
                                   ts.anyType);
 }
 
+function convertIndexableObject(
+    node: doctrine.type.TypeApplication,
+    templateTypes: string[]): ts.IndexableObjectType|ts.NameType {
+  if (node.applications.length !== 2) {
+    console.error('Parameterized Object must have two parameters.');
```

Comment:
maybe assert and/or throw since this should be an internal error?

Reply:
It's not an internal error, actually there is a case in Polymer where a function was annotated with `Object<Node>` which isn't valid AFAIK.